tsdb/engine/tsm1/cache.go

```diff
@@ -67,8 +68,17 @@ func (e *entry) deduplicate() {
 
 // Statistics gathered by the Cache.
 const (
-	statCacheMemoryBytes = "memBytes"  // Size of in-memory cache in bytes
-	statCacheDiskBytes   = "diskBytes" // Size of on-disk snapshots in bytes
+	// levels - point in time measures
+
+	statCacheMemoryBytes = "memBytes"      // level: Size of in-memory cache in bytes
+	statCacheDiskBytes   = "diskBytes"     // level: Size of on-disk snapshots in bytes
+	statSnapshots        = "snapshotCount" // level: Number of active snapshots.
+	statCacheAge         = "cacheAge"      // level: milliseconds since cache was last snapshotted at sample time
+
+	// counters - accumulative measures
+
+	statCachedBytes       = "cachedBytes"       // counter: Total number of bytes written into snapshots.
+	statWALCompactionTime = "WALCompactionTime" // counter: total number of milliseconds spent compacting snapshots
```

Comment:
Would you mind adding the units to the stat names? i.e. `WALCompactionTimeMs` and `cacheAgeMs`.

The comments you've added (and your PR notes as well) are very clear and informative, but having the unit in the name of the stat eliminates some potential guesswork.